cli/cmd/check.go

```diff
@@ -87,7 +90,9 @@ func configureAndRunChecks(w io.Writer, options *checkOptions) error {
 		} else {
 			checks = append(checks, healthcheck.LinkerdPreInstallClusterChecks)
 		}
-		checks = append(checks, healthcheck.LinkerdPreInstallChecks)
+		if !options.cniEnabled {
+			checks = append(checks, healthcheck.LinkerdPreInstallCapabilityChecks)
```

Comment:
Ah, sorry, this was a result of the last change I asked you to make on this PR, but it looks like you're removing `healthcheck.LinkerdPreInstallChecks` altogether here. If you restore line 90, that should fix it.

Reply:
oof nice catch! will push a new PR shortly